### DISCUSSION TITLE
Additional button "Edit" the entry

### DISCUSSION THREAD

**Discussion by LenardHerzogDWB** [khoait/DCE.PCF#103] [Ideas]
Hi Khoa, 
I'm Lenard from Germany. And my question is:
Is it possible to add another button for changing the entry, like the "quickcreate" button?

![Snag_100823ab](https://github.com/khoait/DCE.PCF/assets/141009497/ca801052-0c47-441c-814d-1cbf11ff05e9)

best regards 
Lenard

**Comment by khoait** [comment]
Hi Lenard, thanks for your suggestion. Do you mean edit each tag entry? I'm thinking about clicking on the tag and open a quick edit form as a dialog instead of an edit button in the popup. What do you think?

![image](https://github.com/khoait/DCE.PCF/assets/8686497/7fa166bc-187c-4c43-9746-1e6d950618af)

**Reply by LenardHerzogDWB** [reply]
Hi Khoa and thanks so much for the answer!
Your idea is even better! But would it also be possible to only change the entry if I am the owner?
Best regards Lenard

**Reply by khoait** [reply]
Yes, but to keep it generic for everyone, we can only respect the table permission, not just the owner field. So when users have Write privilege at necessary level, can edit, otherwise the form is read-only.

**Reply by LenardHerzogDWB** [reply]
Ok, that's the right way. I will adjust the permissions.

**Reply by khoait** [reply]
cool. I'll try to implement this in the next release maybe this month. thanks!

**Reply by LenardHerzogDWB** [reply]
great!! Thank you and I'm excited! Could you inform me or should I observe myself?
best regards 🌻Lenard

**Reply by khoait** [reply]
I'll let you know when it's ready. Cheers!

**Reply by khoait** [reply]
Hi @LenardHerzogDWB, happy to share that this is done in the latest release: https://github.com/khoait/DCE.PCF/releases/tag/v1.0.6.0

Please check it out and feel free to open new issues if any. Cheers

**Reply by LenardHerzogDWB** [reply]
Hi Koha, you are the best! Thank you so much for this very quick implementation. Your solution works fine.
Thanks and have a good day! 
🌻Lenard
